packages/framework-types/src/envelope.ts

```diff
@@ -77,6 +78,7 @@ export interface ConnectionDataEnvelope {
 }
 
 export interface UserEnvelope {
+  id?: string
```

Comment:
Why optional? I think the `sub` claim is always in the token.

Asking this because it would make the user code a bit messier, as they would need to _always_ check if the id is not null, while it will never be (if the UserEnvelope is not null, then the id will always be there)

Reply:
Mainly for one reason @alvaroloes, we still have some code specially in the `auth-adapter.ts` which is still using the `CognitoUserPoolEvent` to re-create the UserEnvelope. In those cases I don't know if that object contains the userId. In next iterations I will fix that.